README.md

```diff
@@ -1 +1,14 @@
-# malden-Personal-Project-1
\ No newline at end of file
+# Project Proposal
+High Score Tracker
+[ ] List: txt w/ all current scores
+[ ] Player input: name and score
+[ ] Loop: checking where player lies in all scores
+    - If letters in score, say "invalid input" 
+
+
+## Overview
+My program's purpose is to be a high score tracker. It takes the player's input and compares the player's high score to other high scores inputted, and based on the score inputs the score in order of score size (?). It then displays the list of scores from highest score to lowest score. The scores are stored in a text file. 
+## High-level flowchart
+![Overview](Overview.png)
```

Comment:
Your method flowcharts look great. However, you are missing a method signature, a comment for each of your methods, and a step by step English algorithm.

From the instructions:

```
For each method flowchart, come up with a method signature and a comment. Use the following questions as a guide:

What data do I need to pass to this method?
What are valid inputs?
What does the method accomplish?
What (if any) does this method return (produce)?
At a high level, what does this method do?
Based on #5, what is a short name I can give this method?
Finally, in English, write a step-by-step detailed explanation describing how this method will be implemented.
```

You are welcome to copy the comments / algorithms from the proposal template as long as you feel they match.